api/v1/groupversion_info.go

```diff
@@ -12,10 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package v1beta1 contains API Schema definitions for the app v1beta1 API group
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Why is this second licence added here?

Reply:
not sure, I had removed second licence manually